src/platforms/javascript/common/install/loader.mdx

```diff
@@ -0,0 +1,234 @@
+
+If Performance Monitoring and/or Session Replay is enabled, we immediately need to fetch and initialize the bundle to make sure we are able to capture transactions/replays when the page loaded.
+
+## SDK Configuration
+
+You can configure the release by adding the following to your page:
+
+```html
+<script>
+  window.SENTRY_RELEASE = {
+    id: "..."
+  };  
+</script>
+```
+
+The loader script always includes a call to `Sentry.init` with some default configuration, including your DSN. If you want to [configure your SDK](../../configuration/options/) beyond that, you'll need a second script tag, in which you'll call `Sentry.onLoad`. This script must come _after_ the main loader script.
+
+```html
+<script
+  src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
+  crossorigin="anonymous"
+></script>
+<script>
+  Sentry.onLoad(function() { ... });
+</script>
+```
+
+`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set.
+
+```html
+<script>
+  Sentry.onLoad(function() {
+    Sentry.init({
+      release: " ... ",
+      environment: " ... "
+    });
+    Sentry.configureScope(scope => {
+      scope.setTag( ... );
+    });
+    // etc.
+  });
+</script>
+```
+
+For Performance Monitoring, by default the SDK will be initialized with `tracesSampleRate: 1` and for Session Replay `replaysSessionSampleRate: 0.1` and `replaysOnErrorSampleRate: 1`. Put it words, we capture all pages with Performance Monitoring, Replays of 10% of all sessions as well as Replays for all sessions with an error.
+
+## **Limitations for error-only capturing**
+
+Because the loader injects the SDK asynchronously (only for errors), only _unhandled errors_ and _unhandled promise rejections_ will be caught and buffered before the SDK is fully loaded. Specifically, the capturing of any [breadcrumb data](../../enriching-events/breadcrumbs/) will not be available until the SDK is fully loaded and initialized. To reduce the amount of time these features are unavailable, set `data-lazy="no"` or call `forceLoad()` as described above.
+
+If you want to understand the inner workings of the loader itself, you can read the documented source code in all its glory over at the [sentry repository](https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/js-sdk-loader.ts).
+
+# CDN
```

Comment:
The headers in the rest of this doc are all 1 layer too shallow, which means they're broken in the "on this page sidebar".

My top choice would be to keep this CDN content in it's own page, but if not, all these headers need to be 1 layer more nested, e.g. 

```suggestion
## CDN
```

Reply:
The idea to put this there is to hide this away - we want people to use the loader, not the CDN!